perun-dispatcher/src/main/java/cz/metacentrum/perun/dispatcher/scheduling/impl/TaskSchedulerImpl.java

```diff
@@ -111,33 +116,71 @@ public void scheduleTask(Task task) {
 				log.error("Task "
 						+ task.toString()
 						+ " has no engine assigned and there are no engines registered...");
-				return;
+				return true;
 			}
 		}
 
 		log.debug("Facility to be processed: " + facility.getId()
 				+ ", ExecService to be processed: " + execService.getId());
+		
 
-		log.debug("Is the execService ID:" + execService.getId() + " enabled globally?");
-		if (execService.isEnabled()) {
-			log.debug("   Yes, it is globally enabled.");
-		} else {
-			log.debug("   No, execService ID: "+ execService.getId() + " is not enabled globally. Task will not run.");
-			return;
+		Boolean abortTask = false;
+		try {
+			refetchTaskInformation(task);
+			List<Service> assignedServices = perun.getServicesManager().getAssignedServices(perunSession, task.getFacility());
```

Comment:
There is no bean `PerunBl`, there is only `perun`, typecast using Spring mechanism for getting beans (not `@Autowire`). I used that, though personally I feel it is a kind of hack.